libr/util/pj.c

```diff
@@ -194,6 +202,14 @@ R_API PJ *pj_n(PJ *j, ut64 n) {
 	return j;
 }
 
+R_API PJ *pj_sn(PJ *j, st64 n) {
```

Comment:
why is this required? the comma should be automatically a dded if the parent element is an array or object. and the raw thing is pj_kd.. ..

ok so u want to represent st64? pj_sn is a bad name for this (signed number) because all 2 letters apis are KEY, VALUE, so its confusing, it should be a single letter, also the meaning of pj_s is misleading this function name. maybe pj_N instead?

Reply:
i just copy-pastaed this from the things below. I'm not 100% sure why this comma thingy is required.
Yeah naming is an issue, i'll fix that.

I don't get what you mean by "the raw thing is pj_kd" from what i see this is for vars of type double?